src/ui/components/mixins/notebook-snapshot.js

```diff
@@ -0,0 +1,114 @@
+import $ from 'zepto';
```

Comment:
We don't need to import Zepto here. Vanilla JavaScript will cover all of the cases zepto is used for here.

Reply:
removed